### ISSUE TITLE
Failure to linkify

### ISSUE THREAD

**Issue by sheldrake** [AKASHAorg/akasha-world-framework#1463]
**Describe the bug**
When I pasted a URL including a "?" and subsequent variables, it wasn't linkified, so to speak.

**To Reproduce**
1. Copy a URL with a "?" and variables
2. Paste it into a post
3. Publish

**Expected behavior**
I would expect the link to turn blue indicating that the editor recognises it as a URL, hyperlinking it accordingly.

**Screenshots**
Per https://akasha.ethereum.world/social-app/post/01fk0c54w858r7mp4dyg88r5r5 
![Screenshot 2021-10-27 at 11 49 53](https://user-images.githubusercontent.com/208643/139042718-ee9d2602-0daa-400f-86c0-ff86ef469a69.png)

**Desktop (please complete the following information):**
Mac OS 11.6
FF 93

**Comment by SeverS** [comment]
Hey @sheldrake , did you copied the whole text? eg: "Just found https://....." ?
I cannot reproduce it when pasting just a link.

**Comment by sheldrake** [comment]
Hi @SeverS ... I typed "I just found " into the editor, and then ctrl-v to paste in the URL. Let me try again.

OK, I can update the way to repeat the problem.

It seems I may have had an additional space at the end of the URL. I've just tried pasting a number of URLs into the editor ending in a space, and none of them get linked.

I have no idea how a space may have been added to the URL when I just copied the URL from the browser bar.
